lib/wings/active_fedora_classifier.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Wings
+  class ActiveFedoraClassifier < ActiveFedora::ModelClassifier
+    private
+
+      def classify(model_value)
+        match = model_value.match(/Wings\((.*)\)/)
+        match ? Wings::ActiveFedoraConverter::DefaultWork(match[1].constantize) : super
```

Comment:
I think this would help connect the commit for me:

```ruby
if match
  klass = match[1].constantize
  Wings::ActiveFedoraConverter::DefaultWork(klass)
else
  super
end
```

A part of me wants `DefaultWork(for: klass)` but that is a possible idiomatic shift. Some of which it is still somewhat atypical to see method names that are not lower case (even though Ruby has `Array()`)

Reply:
Hyrax has been following the Dry.rb style of `ModuleName()` and `ClassName()` for module and class builder methods; see, e.g. `Hyrax::ChangeSet()` and `Hyrax::Schema()`, so that's the approach followed here.

i like your rephrase above though.

Reply:
Yeah. I can certainly get behind method names such as these. Just trying to adjust 15 years of Ruby as informed by Rails.